### DISCUSSION TITLE
[Feature] share library across account

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#4460] [Feature Request]
### The feature

Is it possible to share libraries that points to the same path on disk across accounts?
User only need to sync once and all accounts will get updated automatically.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by AlliotTech** [comment]
I think so!

**Comment by nodis** [comment]
me too

**Comment by ghost** [comment]
supported in the latest version
